src/main/java/thestonedturtle/partypanel/PartyPanel.java

```diff
@@ -128,4 +112,21 @@ void removePartyPlayer(final PartyPlayer player)
 			renderSidebar();
 		}
 	}
+
+	void updatePartyMembersExpand(boolean expand)
+	{
+		for (PlayerPanel panel : playerPanelMap.values())
+		{
+			panel.setShowInfo(expand);
+			if (expand)
```

Comment:
Just pass `expand` to the `setExpandIcon`

Reply:
Oh yeah that was a silly over-complication I did. Good catch.